src/v2/styles/Message/Message-layout.scss

```
@@ -59,12 +62,21 @@
   }
 }
 
+.str-chat__quoted-message-txt {
```

I would avoid omitting one letter as it would not save a lot of effort writing it, but could lead to misunderstanding.
```suggestion
.str-chat__quoted-message-text {
```

---

The issue here is that we already have `str-chat__quoted-message-text` and I couldn't find a better name for this class

---

Can it be a variation of the text wrapping element (`.str-chat__quoted-message-text--clamped`) or is it a completely different element altogether?

---

I wanted to add meaningful class names to integrators in case they want to target the element that contains the message text, due to `message-text` already being used I added:
- `str-chat__quoted-message-txt` -> for quoted messages, and added out-of-the-box clamping there, as most chat apps seem to be doing clamping there
- `str-chat__message-txt` -> for non-quoted messages, integrators who want to implement clamping with customization can use that class (as I don't plan to add this as an SDK feature for non-quoted messages)

So while `str-chat__quoted-message-text--clamped` is a nice name for what it does in the SDK, it's not a good name for integrators who want to add custom CSS to this element.

But I agree that `txt` is not an optimal name. Do you like `str-chat__message-text-element` and `str-chat__quoted-message-text-element` better?

---

Or `message-text-value`?

---

`str-chat__message-text-value` and `str-chat__quoted-message-text-value` also works for me

---

@arnautov-anton WDYT?

---

Ok, let met update to `-text-value`

---

All PRs updated :)